sample-configs/podpools-policy.cfg

```diff
@@ -0,0 +1,101 @@
+# This example demonstrates pod-based CPU and memory pinning.
+# All containers of a pod run in the same CPU/memory pool.
+# The capacity of a pool is defined as a number of pods it can
+# contain.
+#
+# You can launch a pod to a pod pool by annotating it:
+#
+# metadata:
+#   annotations:
+#     pool.podpools.cri-resource-manager.intel.com: singlecpu
+
+policy:
+  # pod-based CPU and memory pinning is implemented in the podpools policy.
+  Active: podpools
+
+  # AvailableResources specifies CPUs that the policy is allowed to manage.
+  # By default AvailableResources contains all CPUs.
+  AvailableResources:
+    # In this example we use 14 CPUs, excluding CPUs #0 and #1
+    # (hyperthreads of core 0).
+    CPU: cpuset:2-15
+
+  # ReservedResources specifies CPU(s) that the policy dedicates for
+  # running kube-system pods.
+  ReservedResources:
+    # Here we dedicate CPU #15 for these pods.
+    # This leaves 13 out of 14 available CPUs unallocated.
+    CPU: cpuset:15
```

Comment:
Maybe we could mention here what are the correct corresponding system- and kube-(CPU-)reservation flags to be passed to `kubelet`.

Reply:
Added.